app/views/hosts/edit.html.erb

```diff
@@ -1,4 +1,4 @@
-<% title(_("Edit %s") % @host) %>
+<% title(_("Edit %s") % "#{Setting[:append_domain_name_for_hosts] ? @host.name : @host.shortname}") %>
```

Comment:
Can we have some helper method on the host object? Or isn't this equal to `to_label`?